### DISCUSSION TITLE
How to sign messages using stdlib?

### DISCUSSION THREAD

**Discussion by hmzdot** [reach-sh/reach-lang#1003] [Q&A] [answered]
Aman's question:
> Hi guys, is there any way to sign a message using wallet connect and reach on algorand? I want to authenticate that user on a centralized server and generate access tokens. So that they can access the centralized backend service and make authenticated calls.

Question can extend to "How to sign messages using stdlib?". I know it is possible to import `algosdk` and call signBytes with private key directly but not sure how it is possible using wallets

**Comment by jeapostrophe** [comment, marked as answer]
Wallets on Algorand don't support signing arbitrary data. It is not part of the WalletConnect standard, Pera Wallet app, or MyAlgo Wallet. So, there's no API to use to implement this feature.
